### ISSUE TITLE
API v2.1 not fully evolved

### ISSUE THREAD

**Issue by jotoeri** [nextcloud/forms#1538]
We added some routes for API v2.1, but forgot the PR to adapt all other routes.

**Comment by Chartman123** [comment]
@jotoeri should we move all current APIs to v2(.1) and keep the new ones on v2.1? And what to do with the other PRs that will extend our API? Should we then move on to v2.2?

**Comment by jotoeri** [comment]
> should we move all current APIs to v2(.1) and keep the new ones on v2.1? 

Yes, this should be it. And adding 2.1 to the capabilities and adapting the docs (just replacing 2 with 2.1, the new routes should be documented by the PRs iirc).

> Should we then move on to v2.2?
Yes - it's just again, that routes can be missing on some instances.
